styles.css

```diff
@@ -0,0 +1,841 @@
+  font-family: Inter;
+  line-height: 27px;
+  padding-top: 24px;
+}
+
+.contact {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(2, 130px);
+  gap: 20px;
+}
+@media (min-width: 744px) {
+  .contact {
+    grid-template-columns: repeat(6, 80px);
+    gap: 24px;
+  }
+}
+@media (min-width: 1260px) {
+  .contact {
+    grid-template-columns: repeat(12, 63px);
+    gap: 24px;
+  }
+}
+.contact__form {
+  grid-column: span 2;
+}
+@media (min-width: 744px) {
+  .contact__form {
+    grid-column: span 3;
+  }
+}
+@media (min-width: 1260px) {
+  .contact__form {
+    grid-column: 1/6;
+  }
+}
+.contact__address {
+  grid-column: span 2;
+}
+@media (min-width: 744px) {
+  .contact__address {
+    grid-column: span 3;
+  }
+}
+@media (min-width: 1260px) {
+  .contact__address {
+    grid-column: -3/-6;
+  }
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+.form__input {
+  background: #f8f8fa;
+  border-radius: 40px;
+  border: 1px solid #f8f8fa;
+  width: 100%;
+  height: 48px;
+  line-height: 48px;
+  padding-left: 17px;
+  box-sizing: border-box;
+}
+.form__input::placeholder {
+  font-family: "Inter";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  color: #7E7E83;
+}
+.form__input:hover {
+  border: 1px solid #D5D7DE;
+  box-sizing: border-box;
+}
+.form__input:hover::placeholder {
+  color: #292929;
+}
+.form__input:focus {
+  outline: 1px solid #292929;
+}
+.form__textarea {
+  background: #f8f8fa;
+  border-radius: 16px;
+  border: 0;
+  width: 100%;
+  padding-left: 17px;
+  padding-top: 14px;
+  box-sizing: border-box;
+  height: 144px;
+  font-size: 14px;
+  line-height: 20px;
+}
+.form__textarea::placeholder {
+  font-family: "Inter";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  color: #7E7E83;
+}
+.form__textarea:focus {
+  outline: 1px solid #292929;
+}
+.form__label:not(:focus):not(:active) {
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
+.form__item--message {
+  padding-bottom: 16px;
+}
+
+.address__label {
+  color: #9FADC2;
+  font-size: 14px;
+  font-family: Inter;
+  line-height: 140%;
+}
+.address__content {
+  font-size: 16px;
+  font-family: Inter;
+  line-height: 140%;
+  padding: 8px 0 24px;
+}
+.address__content a {
+  text-decoration: none;
+  color: #3E3E3E;
+}
+
+.footer {
+  height: 200px;
+  background-image: url(./src/images/photos/footer.jpg);
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+  background-size: cover;
+  background-position: center;
+}
+@media (min-width: 744px) {
+  .footer {
+    height: 400px;
+  }
+}
+@media (min-width: 1260px) {
+  .footer {
+    height: 460px;
+  }
+}
+
+.icons__burger {
+  padding-left: 24px;
+}
+@media (min-width: 744px) {
+  .icons__burger {
+    padding-left: 32px;
+  }
+}
+
+/*# sourceMappingURL=styles.css.map */
```

Comment:
It's recommended to remove the source mapping URL in production, as it's only useful during development.